inference-engine/src/readers/onnx_reader/ie_onnx_reader.cpp

```diff
@@ -17,52 +16,26 @@ std::string readPathFromStream(std::istream& stream) {
     // read saved path from extensible array
     return std::string{static_cast<char*>(stream.pword(0))};
 }
-
-/**
- * This helper struct uses RAII to rewind/reset the stream so that it points to the beginning
- * of the underlying resource (string, file, ...). It works similarily to std::lock_guard
- * which releases a mutex upon destruction.
- *
- * This makes sure that the stream is always reset (exception, successful and unsuccessful
- * model validation).
- */
-struct StreamRewinder {
-    StreamRewinder(std::istream& stream) : m_stream(stream) {
-        m_stream.seekg(0, m_stream.beg);
-    }
-    ~StreamRewinder() {
-        m_stream.seekg(0, m_stream.beg);
-    }
-private:
-    std::istream& m_stream;
-};
 } // namespace
 
-bool ONNXReader::supportModel(std::istream& model) const {
-    StreamRewinder rwd{model};
-
-    const auto model_path = readPathFromStream(model);
-
-    // this might mean that the model is loaded from a string in memory
-    // let's try to figure out if it's any of the supported formats
-    if (model_path.empty()) {
-        if (!is_valid_model(model, onnx_format{})) {
-            model.seekg(0, model.beg);
-            return is_valid_model(model, prototxt_format{});
-        } else {
-            return true;
-        }
-    }
+ONNXReader::ONNXReader()
```

Comment:
Can we remove ONNX reader and use Frontend instead?

Reply:
done